(初めてのJavaプログラム。) (画面に出力しています。)という文の最後にセミコロンが欠如しているとどうなるか。という演習で、プログラム…

sekiguchishuntaro opened the pull request:
…をコンパイルした結果、エラー表示という結果でした。確認お願いします。

aaaabc committed:
(初めてのJavaプログラム。) (画面に出力しています。)という文の最後にセミコロンが欠如しているとどうなるか。という演習で、プログラムをコンパイルした結果、エラー表示という結果でした。確認お願いします。

ddt-mentor commented on ensyu1_1/src/ensyu1_1/ensyu1_1.java:
```diff
@@ -0,0 +1,10 @@
+package ensyu1_1;
+
+public class ensyu1_1 {
```
1.　クラスヘッダにコメントを記載してください。
2. 　関数ヘッダにコメントを記載してください。
3. 　1処理ごとにコメントを記載してください(何をしているかが分かるような)。
※研修のしおり 4.1コメントについて 参照

aaaabc committed:
1.　クラスヘッダにコメントを記載してください。
2. 　関数ヘッダにコメントを記載してください。
3. 　1処理ごとにコメントを記載してください(何をしているかが分かるような)。
3つの指摘がございましたので、修正致しました。
ご確認お願いします。

aaaabc committed:
演習1_1の修正を行いました。
ご確認お願いします。

ddt-mentor commented on ensyu1_1/src/ensyu1_1/ensyu1_1.java:
```diff
@@ -2,17 +2,17 @@
+	 * 作成日：2023.04.04
 	 */
 	public static void main(String[] args) {
 	//初めてのJavaプログラムと表示されるようにしている。	
```
コメントのインデントは、原則コード位置と合わせてください。

該当行：18行目、20行目

aaaabc committed:
遅れて申し訳ありません。修正行いました。
